.gitmodules

```diff
@@ -1,6 +1,6 @@
 [submodule "lvgl"]
 	path = lvgl
-	url = https://github.com/littlevgl/lvgl.git
+	url = https://github.com/lgl/lvgl.git
```

Comment:
How did the test pass??
You have a typo: github.com/**lgl**/lvgl.git

Reply:
Submodules are updated from `lv_bindings` *before* the PR copy is checked out, so it uses whatever is on GitHub. That looks like an oversight in the CI.